pkg/kudoctl/cmd/init.go

```diff
@@ -143,36 +143,36 @@ func (initCmd *initCmd) run() error {
 	//define an Encoder to replace YAMLWriter
 	if strings.ToLower(initCmd.output) == "yaml" {
 
-		var mans []string
+		var manifests []string
 
 		crd, err := cmdInit.CRDs().AsYaml()
 		if err != nil {
 			return err
 		}
-		mans = append(mans, crd...)
+		manifests = append(manifests, crd...)
 
 		if !initCmd.crdOnly {
 			prereq, err := cmdInit.PrereqManifests(opts)
 			if err != nil {
 				return err
 			}
-			mans = append(mans, prereq...)
+			manifests = append(manifests, prereq...)
 
 			if len(opts.Webhooks) != 0 { // right now there's only 0 or 1 webhook, so this is good enough
-				prereq, err := cmdInit.WebhookManifests(opts.Namespace)
+				webhooks, err := cmdInit.WebhookManifests(opts.Namespace)
```

Comment:
Nit, though it's already better than `prereq`:
```suggestion
				webhookManifests, err := cmdInit.WebhookManifests(opts.Namespace)
```

Reply:
uf that is almost too long for me :)

Reply:
Waaay too long! Let's just accept the fact that Go has no [Getters and Setters](https://golang.org/doc/effective_go.html#Getters) and intentionally chooses [small variable names](https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md#variables), especially in the local scope.